docs/details/commands/pmm-admin.md

```diff
@@ -325,10 +325,28 @@ When you remove a service, collected data remains on PMM Server for the specifie
 
 ##### Advanced Options
 
-###### Enable all collectors
-
 PMM starts the MongoDB exporter by default only with `diagnosticdata` and `replicasetstatus` collectors enabled.
 
+    FLAGS:
+
+    `--enable-all-collectors`
+    :  Enable all collectors
+
+    `--disable-collectors`
+    :  Comma-separated list of collector names to exclude from exporter
+
+    `--max-collections-limit=-1`
+    :  Disable collstats, dbstats, topmetrics and indexstats if there are more than <n> collections. 0: No limit. Default is -1, PMM automatically sets this value.
+
+        !!! caution ""
+            Too high limit of `max-collections-limit` could impact the CPU and Memory usage. Check `--stats-collections` to limit the scope of collections and DB's metrics to be fetched.
```

Comment:
Could rephrase as:

A very high limit of `max-collections-limit` could impact the CPU and Memory usage. Check `--stats-collections` to limit the scope of collections and DB's metrics to be fetched.

Reply:
```suggestion
            A very high limit of `max-collections-limit` could impact the CPU and Memory usage. Check `--stats-collections` to limit the scope of collections and DB's metrics to be fetched.
```